tests/lit-tests/vectorcall-3.ispc

```diff
@@ -0,0 +1,34 @@
+// RUN: %{ispc}  %s --emit-asm --target=avx2-i32x8 --disable-vectorcall -o - | FileCheck %s -check-prefix=CHECK_AVX2
```

Comment:
Comment on the intent is needed. Especially, given that you are checking very specific code sequence.

Reply:
I would have settled for a more simple check if I could use something like CHECK-NEXT-NOT to make sure the first instruction is not a mov. If you think the check is too specific, I can just check for a mov

Reply:
I like your idea. Checking that no "mov" are generated is actually what we need here.
This way you can simplify the check and make it universal for any target.

Reply:
In this case, it would be a check to ensure a 'mov' is created for windows when vectorcall is used. I've made the change.